content/pytorch/concepts/tensors/terms/tensor/view/view.md

```diff
@@ -0,0 +1,162 @@
+  - 'Data Types'
+  - 'Deep Learning'
+  - 'Functions'
+  - 'Math operations'
+  - 'Data Science'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'py-torch-for-classification'
+---
+--------------
+
+# Tensor.view(*shape)
+Returns a new tensor with the same data as the self tensor but of a different shape.
+
+The **`.view()`** method in PyTorch is used to reshape the dimensions of a tensor without changing the underlying data. The `.view()` method returns a new tensor with the same number of elements as the original tensor but with a different shape. The `.view()` method for a Tensor object in PyTorch takes the arguments (subspaces), for example, r and c, to reshape the tensor into a matrix with r rows and c columns. The first argument specifies the desired number of rows and the second argument specifies the desired number of columns. The `.view()` method takes in one or more integers as arguments, which specify the desired shape of the tensor. The arguments can be a single integer, in which case the tensor is reshaped into a 1−dimensional tensor with the specified length. Setting -1 as the 1st number can adjust the size automatically so you don't need to set 12, 2, 3, 4 or 6 as the 1st number. *-1 is available only as the 1st number. Alternatively, the arguments can be a tuple of integers, in which case the tensor is reshaped into a tensor with the specified shape.
+
+## Syntax
+
+shape (torch.Size or int...) – the desired size
+
+## Examples
+
+```python
+# importing torch module
+    import torch
+# create a one-dimensional tensor with 10 elements
+    a = torch.FloatTensor([10, 20, 30, 40, 50, 1, 2, 3, 4, 5])
+# view the tensor in 5 rows and 2 columns
+    print(a.view(5, 2))
+    
+    tensor([[10., 20.],
+            [30., 40.],
+            [50.,  1.],
+            [ 2.,  3.],
+            [ 4.,  5.]])
+# view the tensor in 2 rows and 5 columns
+    print(a.view(2, 5))
+    
+    tensor([[10., 20., 30., 40., 50.],
+            [ 1.,  2.,  3.,  4.,  5.]])
+```
+
+```python
+# create a two dimensions tensor 16 elements
+    x = torch.randn(4, 4)
+    x.size()
+        torch.Size([4, 4])
+# view the tensor in 16 rows and create a new tensor
+    y = x.view(16)
+    y.size()
+        torch.Size([16])
+# view the tensor with 8 columns and create a new tensor
+    z = x.view(-1, 8)  # the size -1 is inferred from other dimensions
+    z.size()
+        torch.Size([2, 8])
+```
+
+```python
+
+# create a four dimensions tensor
+    a = torch.randn(1, 2, 3, 4)
+    a.size()
+        torch.Size([1, 2, 3, 4])
+# Swaps 2nd and 3rd dimension and create a new tensor "b"      
+    b = a.transpose(1, 2)  
+    b.size()
+        torch.Size([1, 3, 2, 4])
+# View the tensor swapping 2nd and 3rd dimension and create a new tensor "c"        
+    c = a.view(1, 3, 2, 4)  # Does not change tensor layout in memory
+    c.size()
+        torch.Size([1, 3, 2, 4])
+# Compare "b" and "c" tensors        
+    torch.equal(b, c)
+        False
+```
```

Comment:
Wrap all the three different code blocks in one. Do not comment/show the output in the code, you can showcase the output using the ```shell``` block after the code.
and wrap the code in the ```py``` block instead of ```python```